use-cases/task-assigning/src/main/java/org/acme/taskassigning/domain/TaskAssigningSolution.java

```diff
@@ -0,0 +1,88 @@
+package org.acme.taskassigning.domain;
+
+import java.util.List;
+
+import ai.timefold.solver.core.api.domain.solution.PlanningEntityCollectionProperty;
+import ai.timefold.solver.core.api.domain.solution.PlanningScore;
+import ai.timefold.solver.core.api.domain.solution.PlanningSolution;
+import ai.timefold.solver.core.api.domain.solution.ProblemFactCollectionProperty;
+import ai.timefold.solver.core.api.domain.valuerange.ValueRangeProvider;
+import ai.timefold.solver.core.api.score.buildin.bendable.BendableScore;
+import ai.timefold.solver.core.api.solver.SolverStatus;
+
+@PlanningSolution
+public class TaskAssigningSolution {
+
+    private List<TaskType> taskTypes;
+
+    @ProblemFactCollectionProperty
+    private List<Customer> customers;
+
+    @ValueRangeProvider
+    @PlanningEntityCollectionProperty
+    private List<Task> tasks;
+
+    @PlanningEntityCollectionProperty
+    private List<Employee> employees;
+
+    @PlanningScore(bendableHardLevelsSize = 1, bendableSoftLevelsSize = 5)
```

Comment:
This is a bit crazy. I suggest we spend a bit of time thinking about the constraints and how they should be weighted against each other, and based on that, replace this with a normal `HardMediumSoftScore`.

Reply:
Our bendable score has now one hard level and three soft levels.